utils/frame/frame-utilities-cli/Cargo.toml

```
@@ -0,0 +1,39 @@
+[package]
+name = "frame-utilities-cli"
```

this includes both substrate-client and frame features, as it is the integration layer. Thus the naming convention is to call this `substrate-frame-`...

```suggestion
name = "substrate-frame-utilities-cli"
```